src/plugins/better-navigation/index.ts

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable @typescript-eslint/prefer-nullish-coalescing */
```

Comment:
Why? This file seems to use `||` only for booleans, so this rule should not be triggered.

Reply:
Probably an artifact from when I used `||` for a non-boolean in an earlier version, which I then forgot to remove.

Reply:
Oh wait nvm, this relies on it:
```js
function isCtrlArrowSkippableSymbolMQElem(el?: HTMLElement) {
  return (
    el?.classList.contains("dcg-mq-bracket-container") ||
    el?.classList.contains("dcg-mq-fraction") ||
    el?.classList.contains("dcg-mq-large-operator") ||
    el?.classList.contains("dcg-mq-int") ||
    el?.classList.contains("dcg-mq-sqrt-container") ||
    el?.classList.contains("dcg-mq-nthroot-container")
  );
}
```